packages/blade/src/components/Indicator/_decisions/decisions.md

```diff
@@ -0,0 +1,111 @@
+# Indicator
+
+Indicators describe the condition of an entity. They can be used to convey semantic meaning, such as statuses and semantical-categories.
+
+This document outlines the API of `Indicator` component.
+
+<img src="./indicator-thumbnail.png" width="380" alt="" />
+
+## Design
+
+- [Figma - Indicator](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=8224%3A1)
+
+## API
+
+Sample usecase:
+
+<img src="./with-label.png" width="380" alt="" />
+
+Sample usage:
+
+```jsx
+<Indicator
+  intent="positive"
+  label="Active | xyz@email.com"
+  accessibilityLabel="Active for xyz@email.com"
+/>
+```
+
+### Indicator
+
+We'll expose an `Indicator` component with the following API:
+
+| Prop               | Type                                                      | Default     | Description                                    | Required |
+| ------------------ | --------------------------------------------------------- | ----------- | ---------------------------------------------- | -------- |
+| accessibilityLabel | `string`                                                  | `undefined` | Sets the aria-label for screen readers         | ♿️      |
+| label              | `string`                                                  | `undefined` | Shows a text label beside the circle indicator |          |
+| intent             | `positive`,`negative`, `notice`, `information`, `neutral` | `neutral`   | Sets the color tone of circle indicator        |          |
+| size               | `xsmall`,`small`, `medium`                                | `medium`    | Sets the size of circle indicator              |          |
+
+> **Note**
+>
+> ♿ `accessibilityLabel` is **required** when a `label` isn't provided and helps in setting `aria-label` for people relying on screen readers. Some people with color blindness can't distinguish between colors easily so passing a `label` / `accessibilityLabel` is crucial.
+
+#### Composition with other UI elements
+
+`Indicator` can be clubbed with other UI elements to show a dot at the top-right corner, generally for capturing user attention. Example:
+
+<img src="./with-other-components.png" width="380" alt="" />
+
+Sample usage:
+
+```jsx
+<div style={{ position: 'relative' }}>
+  <NotificationWidget />
+  <div style={{ position: 'absolute', top: 0, right: 0 }}>
+    <Indicator intent="notice" accessibilityLabel="Pending notifications" />
+  </div>
+</div>
+```
+
+> **Note**
+>
+> At the moment we don't provide an out of box API for such composition. See the alternatives section for more info.
+
+## a11y
+
+Web:
+
+- The component accepts an `accessibilityLabel` prop to let users pass an `aria-label` used by screen readers. This is helpful in building UIs where an `Indicator` is used without a visible `label`.
+- If `label` is provided without any `accessibilityLabel` we use an `aria-labelledby` internally to let the screen readers use `label` for a11y purposes.
```

Comment:
> note: depending on the html structure we might not need to set arialabelledby if the content is a immediate children.

Reply:
I think it'll need to be sibling (flex siblings) but will keep this in mind